src/plop.js

```diff
@@ -98,25 +113,32 @@ function run(env) {
 /////
 // everybody to the plop!
 //
-function doThePlop(generator) {
-	generator.runPrompts().then(generator.runActions)
+function doThePlop(generator, bypassArr) {
+	generator.runPrompts(bypassArr)
+		.then(generator.runActions)
 		.then(function (result) {
 			result.changes.forEach(function(line) {
 				console.log(chalk.green('[SUCCESS]'), line.type, line.path);
 			});
 			result.failures.forEach(function (line) {
-				var logs = [chalk.red('[FAILED]')];
+				const logs = [chalk.red('[FAILED]')];
 				if (line.type) { logs.push(line.type); }
 				if (line.path) { logs.push(line.path); }
 
-				var error = line.error || line.message;
+				const error = line.error || line.message;
 				logs.push(chalk.red(error));
 
 				console.log.apply(console, logs);
 			});
 		})
 		.catch(function (err) {
-			console.error(chalk.red('[ERROR]'), err.message, err.stack);
+			console.error(chalk.red('[ERROR]'), err.message);
 			process.exit(1);
 		});
 }
+
+function listHasOptionThatStartsWith(list, prefix) {
+	return list.some(function (txt) {
```

Comment:
I think we can also make this code more concise − but still readable − with an arrow function: `list.some((txt) => txt.indexOf(prefix) === 0)`

Reply:
yeah, this really tore me up inside when I wrote this 😦